samples/kvs_gstreamer_file_uploader_sample.cpp

```diff
@@ -240,20 +238,32 @@ int main(int argc, char* argv[]) {
     int retry_count = DEFAULT_RETRY_COUNT;
     int stream_status = STREAM_STATUS_OK;
     bool do_retry = true;
+    char *ptr;
 
     if (strcmp(argv[1], "--help") == 0 || strcmp(argv[1], "-h") == 0) {
         print_usage(argv[0]);
         return 1;
     }
-    data.file_path = argv[1];
-    data.kvssink_str = build_kvssink_str();
+    string stream_name = argv[1];
+    data.file_path = argv[2];
+    if (access(data.file_path.c_str(), F_OK) == -1) {
+        LOG_ERROR("File not found. Path: %s", data.file_path.c_str());
+        return 1;
+    }
+    unsigned long file_start_time = strtoul(argv[3], &ptr, 10);
```

Comment:
if file_start_time is 0 then use current time?

Reply:
Yep